model.py

```
@@ -1,51 +1,86 @@
-            x = torch.relu(conv_layer(x, edge_index))
-            #x = self.dropout_hidden(x)
-
-        # Aggregation step - since this is a graph level classification, the values in each layer must be made coarse to represent one graph
-        x = global_mean_pool(x, batch)
-
-        outputs = self.output_layers(x) #logits output 
-        return outputs
-
+import torch
+import torch.nn as nn
+import torch.optim as optim
```

Ruff (0.8.2)

`torch.optim` imported but unused

Remove unused import: `torch.optim`

(F401)